src/components/sbb-time-input/sbb-time-input.scss

```diff
@@ -0,0 +1,24 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-time-input-max-width: #{sbb.px-to-rem-build(42)};
```

Comment:
@mcilurzo changed the dimension, as the placeholder was too wide for `HH:MM`
It is now 58px and 65px